ssl/record/rec_layer_s3.c

```diff
@@ -1573,6 +1573,9 @@ int ssl3_read_bytes(SSL *s, int type, int *recvd_type, unsigned char *buf,
             SSLfatal(s, SSL_AD_HANDSHAKE_FAILURE, SSL_F_SSL3_READ_BYTES,
                      SSL_R_NO_RENEGOTIATION);
             return -1;
+        } else if (alert_level == SSL3_AL_WARNING && !is_tls13) {
```

Comment:
Formally speaking you don't have to check for is_tls13, because that case is already handled above, in alert_level == SSL3_AL_FATAL || is_tls13.